Source/WebCore/rendering/RenderListMarker.cpp

```diff
@@ -43,1272 +43,10 @@
 
 namespace WebCore {
 
-// As of June 2021, the Microsoft C++ compiler seems unable to include std::initializer_list in a constexpr.
-#if COMPILER(MSVC)
-#define CONSTEXPR_WITH_MSVC_INITIALIZER_LIST_WORKAROUND const
-#else
-#define CONSTEXPR_WITH_MSVC_INITIALIZER_LIST_WORKAROUND constexpr
-#endif
-
```

Comment:
is this change related?

Reply:
Yes, we're no longer using CONSTEXPR_WITH_MSVC_INITIALIZER_LIST_WORKAROUND throughout the file, so we can remove the definition

Reply:
Nevermind, it seems we won't need that anymore.